ui/src/css/reactable.css

```diff
@@ -5,3 +5,35 @@ th.reactable-header-sort-asc:after {
 th.reactable-header-sort-desc:after {
   content: '\25B2';
 }
+
+tbody.reactable-pagination tr td {
+    text-align: center !important;
+}
+tbody.reactable-pagination tr td > .reactable-current-page {
+    color: #fff;
+    cursor: default;
+    background-color: #337ab7;
+    border-color: #337ab7;
+}
+
+tbody.reactable-pagination tr td > a:first-child {
+    border-top-left-radius: 4px;
+    border-bottom-left-radius: 4px;
+}
+
+tbody.reactable-pagination tr td > a:last-child {
+    border-top-right-radius: 4px;
+    border-bottom-right-radius: 4px;
+    border-right-width: 1px;
+}
+
+tbody.reactable-pagination tr td > a {
+    position: relative;
+    padding: 6px 12px;
+    line-height: 1.42857143;
+    color: $blue;
```

Comment:
We don't have any css pre-processing, so I guess this probably needs to be a hex colour value.